chat-android/src/test/java/com/ably/chat/TestUtils.kt

```diff
@@ -60,7 +60,7 @@ fun mockOccupancyApiResponse(realtimeClientMock: RealtimeClient, response: JsonE
 
 internal class EmptyLogger(override val context: LogContext) : Logger {
     override fun withContext(tag: String?, staticContext: Map<String, String>, dynamicContext: Map<String, () -> String>): Logger = this
-    override fun log(message: String, level: LogLevel, throwable: Throwable?, newTag: String?, newStaticContext: Map<String, String>) = Unit
+    override fun log(message: String, level: LogLevel, throwable: Throwable?, newTag: String?, context: Map<String, String>) = Unit
```

Comment:
can we rename `newTag` as well? I think just `tag` will be better

Reply:
fixed